mapproxy/util/ext/wmsparse/duration.py

```diff
@@ -0,0 +1,586 @@
+"""
+Disclamer: 

```

Comment:
isodate is BSD licensed, so this is OK, but please copy the full header with license.
Also add a note what 'based on' means. Does it only contain parts? Is it just merged into a single file? Are there any changes?